examples/onchain-signer/contracts/Gasless.sol

```diff
@@ -9,22 +9,15 @@ struct EthereumKeypair {
     uint64 nonce;
 }
 
-struct EthTx {
```

Comment:
Should we rename `EIP155Signer.EthTx` to `EIP155Signer.Tx` or `EIP155Signer.Transaction`?